app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivity.kt

```diff
@@ -56,16 +80,16 @@ class AdministratorControlsActivity :
     startActivity(ProfileListActivity.createProfileListActivityIntent(this))
   }
 
-  companion object {
-    fun createAdministratorControlsActivityIntent(context: Context, profileId: Int?): Intent {
-      val intent = Intent(context, AdministratorControlsActivity::class.java)
-      intent.putExtra(NAVIGATION_PROFILE_ID_ARGUMENT_KEY, profileId)
-      return intent
-    }
+  override fun onBackPressed() {
+    val fragment =
+      supportFragmentManager.findFragmentById(
+        R.id.administrator_controls_fragment_multipane_placeholder
+      )
 
-    fun getIntentKey(): String {
-      return NAVIGATION_PROFILE_ID_ARGUMENT_KEY
-    }
+    if (fragment is ProfileEditFragment)
```

Comment:
This needs braces or reformatting per https://developer.android.com/kotlin/style-guide#braces.

Also, this would probably benefit from a line comment explaining this logic since it's not obvious at a glance why it's correct/needed.

Reply:
done